packages/wp-now/README.md

```diff
@@ -34,6 +116,40 @@ To run the unit tests, use the following command:
 nx test wp-now
 ```
 
+### Migrating from Laravel Valet?
+
+If you are migrating from Laravel Valet, you should be aware of the differences it has with `wp-now`:
+
+-   `wp-now` does not require you to install separately WordPress, create database, connect WordPress to that database, create your use or login. All of these steps are packaged into `wp now start` command and are running under the hood;
+-   `wp-now` works across all platforms (Mac, Linux, Windows);
```

Comment:
Did anyone actually test it on Windows? 😅

Reply:
I am not quite sure, I have a Windows machine at home - I can try it later today 😅

Reply:
Quite a few places in wp-now hardcode `/` as a directory separator instead of using `path.join()` but I'm not sure if that's still a problem on modern Windows.